.github/workflows/CI.yml

```
@@ -286,3 +287,13 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo minimal-versions check
       - run: cargo minimal-versions check --features full
+
+  semver:
+    name: semver
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check semver
+        uses: obi1kenobi/cargo-semver-checks-action@v2
+        with:
+          release-type: minor
```

The compile seems to have failed because of `hyper`'s unstable features. It might be fixed by specifying the `full` feature explicitly.

```suggestion
      - name: Check semver
        uses: obi1kenobi/cargo-semver-checks-action@v2
        with:
          release-type: minor
          feature-group: only-explicit-features
          features: full
```